<?xml version="1.0"?>
<launch>
  <arg name="region_origin_x" default="0.0"/>
  <arg name="region_origin_y" default="0.0"/>
  <arg name="search_space_dimension" default="16"/>
  <arg name="search_space_resolution" default="0.3"/>
  <arg name="observation_file" default="$(find movo_object_search)/tmp/obs_info.yaml"/>
  <arg name="action_file" default="$(find movo_object_search)/tmp/action_info.yaml"/>
  <arg name="prior_file" default="$(find movo_object_search)/tmp/prior.yaml"/>
  <arg name="target_object_ids" default="[0,1,2]" />

  <node pkg="movo_object_search" type="search_in_region" name="movo_object_search_in_region">
    <rosparam command="load" file="$(find movo_object_search)/config/yaml/region_search.yaml"/>
    <param name="region_origin_x"          value="$(arg region_origin_x)"/>
    <param name="region_origin_y"          value="$(arg region_origin_y"/>       
    <param name="search_space_dimension"   value="$(arg search_space_dimension)"/>
    <param name="search_space_resolution"  value="$(arg search_space_resolution)"/>
    <param name="observation_file"         value="$(arg observation_file)"/>
    <param name="action_file"              value="$(arg action_file)"/>
    <param name="prior_file"               value="$(arg prior_file)"/>
    <param name="target_object_ids"        value="$(arg target_object_ids)"    type="yaml" />
  </node>
</launch>
